ui/core/components/detailed_results/aura_metrics.ts

```diff
@@ -52,19 +53,25 @@ export class AuraMetricsTable extends MetricsTable<AuraMetrics> {
 			}
 			const player = players[0];
 
-			const auras = player.auras;
+			const auras = this.filterMetrics(player.auras);
 			const actionGroups = AuraMetrics.groupById(auras);
-			const petGroups = player.pets.map(pet => pet.auras);
-
+			const petGroups = player.pets.map(pet => this.filterMetrics(pet.auras));
 			return actionGroups.concat(petGroups);
 		}
 	}
 
 	mergeMetrics(metrics: Array<AuraMetrics>): AuraMetrics {
-		return AuraMetrics.merge(metrics, true, metrics[0].unit?.petActionId || undefined);
+		return AuraMetrics.merge(metrics, {
+			removeTag: true,
+			actionIdOverride: metrics[0].unit?.petActionId || undefined,
+		});
 	}
 
 	shouldCollapse(metric: AuraMetrics): boolean {
 		return !metric.unit?.isPet;
 	}
+
+	filterMetrics(metrics: Array<AuraMetrics>): Array<AuraMetrics> {
+		return metrics.filter(aura => aura.actionId.otherId !== OtherAction.OtherActionMove);
+	}
```

Comment:
Why are you removing the Move metric? I think its important if an encounter has movement in it for the user to be able to see how much time was spent while moving. Or maybe I'm not understanding this and its doing something else

Reply:
You are correct, I need to only remove it when it concerns pet. At least that was the ask.